frontend/src/components/comments/Comments.tsx

```diff
@@ -0,0 +1,74 @@
+import { Typography, Button } from "@mui/material";
+import { useState, useEffect, useMemo } from "react";
+import ExpandLessIcon from "@mui/icons-material/ExpandLess";
+import styled from "@mui/styled-engine-sc";
+import { useAppDispatch, useAppSelector } from "../../store/hooksTyped";
+import { fetchCommentItem } from "../../store/reducer";
+
+const CommentContainer = styled("div")`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  width: 96%;
+`;
+interface CommentIDProp {
+  id: number;
+  getTime: (arg: number) => string;
+}
+
+const CustomButton = styled(Button)`
+  text-align: start;
+  text-transform: none;
+  background-color: #fff;
+`;
+
+function Comments({ id, getTime }: CommentIDProp) {
+  const [expand, setExpand] = useState(false);
+  const commentItem = useAppSelector((state) =>
+    state.newsIds.comments?.find((item) => item.id === id)
+  );
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    dispatch(fetchCommentItem(id));
+  }, []);
+  const getExpand = () => {
```

Comment:
очень сбивает название с толку, мне кажется тут должно быть так
```suggestion
  const toggleExpand = () => {
```

Reply:
исправлено